index.html

```diff
@@ -1 +1,61 @@
+<h1>Email Markup Consortium</h1>
 
+<p>Working to improve the user experience, accessibility, performance,
+consistency, and reliability of email markup</p>
+
+<hr>
+
+<h2>Our Mission</h2>
+
+<p>The Email Markup Consortium (EMC) is a community-led group of industry professionals working to improve the email
+experience for everyone. Recipients should be able to easily access the content of every email sent to them, regardless
+of their device, email client, assistive technology or internet speed. The EMC collaborates with developers, creation
+and sending tools, and email clients to drive this vision forward at every step of the email journey.
+
+<p>Senders, recipients, and email clients share the common goals of improving user experience, accessibility, performance,
+consistency, and reliability. However, currently there is a lack of cooperation in trying to achieve these goals leading
+to inconsistent results. By working together on agreed approaches we can save everyone time, effort, and money while
+delivering better results to the recipients.</p>
+
+<h2>How You Can Help</h2>
+
+<ol>
+  <li>PLACEHOLDER: Share on Twitter, LinkedIn, etc (links with share URLs?)</li>
+  <li>PLACEHOLDER: Join us (How? Link to form, GitHub repo)</li>
+</ol>
+
+<h2>Supporters</h2>
+
+<p>The Email Markup Consortium and its mission are supported by employees of Salesforce, Squarespace, Litmus, Email on Acid, Sparkpost, Constant Contact, ActionRocket, Parcel, and more.</p>
```

Comment:
It would be nice to iterate toward a list of named backers similar to [Overlay Fact Sheet](https://overlayfactsheet.com/#statement-from-sponsors-and-signatories-to-this-fact-sheet).

Reply:
I think relates to the supporters we'd collect on this form
https://docs.google.com/forms/d/e/1FAIpQLSdr0PEwMahURCS6BSg3tMgw5yN6Yz24yJvbHypcn_OK0y94Qw/viewform

Reply:
The "how you can help" section now lists the membership application forms.